menas/ui/service/FilterTreeUtils.js

```diff
@@ -61,6 +61,10 @@ class FilterTreeUtils {
           filterNode.text = "OR";
           filterNode.icon = "sap-icon://split";
           break;
+        case "NotFilter":
```

Comment:
There is no code change, just reordering of cases - now they are in the same order as the `MenuItem`s in the `filterTreeEdit.fragment`. (The different order was most probably the reason for the mixup, on my end)